Std/Lean/CoreM.lean

```diff
@@ -45,5 +45,5 @@ def heartbeatsPercent : CoreM Nat := do
 /-- Log a message if it looks like we ran out of time. -/
 def reportOutOfHeartbeats (tac : Name) (stx : Syntax) (threshold : Nat := 90) : CoreM Unit := do
   if (← heartbeatsPercent) ≥ threshold then
-    logInfoAt stx (s!"`{tac}` stopped because it was running out of time.\n" ++
-      "You may get better results using `set_option maxHeartbeats 0`.")
+    logInfoAt stx s!"`{tac}` stopped because it was running out of time.\n\
+                     You may get better results using `set_option maxHeartbeats 0`."
```

Comment:
```suggestion
    logInfoAt stx s!"\
      `{tac}` stopped because it was running out of time.\n\
      You may get better results using `set_option maxHeartbeats 0`."
```
don't do "visual indent" in std, especially in long strings because this unnecessarily foreshortens the line.